### PR TITLE
Add dach press icons

### PR DESCRIPTION
**Pull request by stewartyu**
Add new SVGs for DACH press icons

Related to https://github.com/CasperSleep/Casper/pull/4561

### CODE REVIEW

**Comment by victorwhy** [comment]
@stewartyu can you minify those?  i use this tool, just upload it up there, check it, and redownload it

https://jakearchibald.github.io/svgomg/

**Comment by ckeisato** [comment]
Can you also add `<title>` and `<desc>` tags?

**Comment by stewartyu** [comment]
@victorwhy Thanks! I'm leaning towards keeping them non-minified since the icon sprite will get minified anyways: https://github.com/CasperSleep/nightshade-icons/blame/master/dist/storefront/sprite.symbol.svg What do you think?

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@casper/nightshade-icons",
-  "version": "2.9.0",
+  "version": "2.9.1",
```
Comment (robwierzbowski) [inline package.json]: Bump to 2.10.0, adding feature instead of bugfix? If we're doing Semver :)

**Comment by victorwhy** [comment]
@stewartyu huh i think i've been minifying everything but you're right that task also minifies everything as well

**Comment by stewartyu** [comment]
Backing out version bump - will do after it's merged.

**Pull request merged**